src/test/app/forms/validation/validators/IsLessThanOrEqualToSumOf.ts

```diff
@@ -0,0 +1,51 @@
+import { expect } from 'chai'
+import { IsLessThanOrEqualToSumOfConstraint } from 'forms/validation/validators/isLessThanOrEqualToSumOf'
+import { ValidationArguments } from 'class-validator'
+
+const DEFAULT_FIELD_TO_SUM_VALUE = 30
+const DEFAULT_VALUE = 200
+
+function validationArgs (fieldToSumValue: number, value: number): ValidationArguments {
+  return {
+    value: undefined,
+    targetName: undefined,
+    object: { fieldToSum: fieldToSumValue, relatedField: value },
+    property: undefined,
+    constraints: ['fieldToSum', 'relatedField']
+  }
+}
+
+describe('IsLessThanOrEqualToSumOfConstraint', () => {
+  const constraint: IsLessThanOrEqualToSumOfConstraint = new IsLessThanOrEqualToSumOfConstraint()
+
+  describe('validate', () => {
+
+    describe('should return true when ', () => {
+      it('given an undefined value', () => {
+        expect(constraint.validate(undefined)).to.equal(true)
```

Comment:
That is interesting, I would have though this to throw an `Invalid number of arguments, got...` error.

Reply:
the first thing it checks is the value... I can change the order if you want?

Reply:
I would have expect validator to be properly configured before it starts its job. That would also help us failing fast. I would recommend switching an order. Worth paying an attention to other validators in the future as well.